sdk/ml/azure-ai-ml/azure/ai/ml/dsl/_pipeline_component_builder.py

```diff
@@ -296,7 +297,11 @@ def _map_type(_meta):
                 name=key,
                 data=None,
                 meta=Output(
-                    type=_map_type(meta), description=meta.description, mode=meta.mode, is_control=meta.is_control
+                    type=_map_type(meta),
```

Comment:
We may have a unified experience for whether promote node level config(mode/is_control/early_available) to pipeline level in SDK side.
Seems currently we will promote is_control and early_available and won't promote mode. 
We can keep current implementation for now, but I think we need a long term task to discuss this.